mkdocs/docs/experiments/categories/gcp/gcp-vm-disk-loss-by-label.md

```diff
@@ -0,0 +1,260 @@
+## Introduction
+
+- It causes chaos to disrupt state of GCP persistent disk volume filtered using a label by detaching it from its VM instance for a certain chaos duration.
+
+!!! tip "Scenario: detach the gcp disk"    
+    ![GCP VM Disk Loss By Label](../../images/gcp-vm-disk-loss.png)
+
+## Uses
+
+??? info "View the uses of the experiment" 
+    coming soon
+
+## Prerequisites
+
+??? info "Verify the prerequisites" 
+    - Ensure that Kubernetes Version > 1.16 
+    -  Ensure that the Litmus Chaos Operator is running by executing <code>kubectl get pods</code> in operator namespace (typically, <code>litmus</code>).If not, install from <a href="https://v1-docs.litmuschaos.io/docs/getstarted/#install-litmus">here</a>
+    -  Ensure that the <code>gcp-vm-disk-loss-by-label</code> experiment resource is available in the cluster by executing <code>kubectl get chaosexperiments</code> in the desired namespace. If not, install from <a href="https://hub.litmuschaos.io/api/chaos/master?file=charts/gcp/gcp-vm-disk-loss-by-label/experiment.yaml">here</a>
+    - Ensure that your service account has an editor access or owner access for the GCP project.
+    - Ensure that the target disk volume is not a boot disk of any VM instance.
+    - Ensure to create a Kubernetes secret having the GCP service account credentials in the default namespace. A sample secret file looks like:
+
+        ```yaml
+        apiVersion: v1
+        kind: Secret
+        metadata:
+          name: cloud-secret
+        type: Opaque
+        stringData:
+          type: 
+          project_id: 
+          private_key_id: 
+          private_key: 
+          client_email: 
+          client_id: 
+          auth_uri: 
+          token_uri: 
+          auth_provider_x509_cert_url: 
+          client_x509_cert_url: 
+        ``` 
+    
+## Default Validations
+
+??? info "View the default validations" 
+    - Disk volumes are attached to their respective instances
+
+## Minimal RBAC configuration example (optional)
+
+!!! tip "NOTE"   
+    If you are using this experiment as part of a litmus workflow scheduled constructed & executed from chaos-center, then you may be making use of the [litmus-admin](https://litmuschaos.github.io/litmus/litmus-admin-rbac.yaml) RBAC, which is pre installed in the cluster as part of the agent setup.
+
+    ??? note "View the Minimal RBAC permissions"
+
+        [embedmd]:# (https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/gcp/gcp-vm-disk-loss-by-label/rbac.yaml yaml)
+        ```yaml
+        ---
+        apiVersion: v1
+        kind: ServiceAccount
+        metadata:
+          name: gcp-vm-disk-loss-by-label-sa
+          namespace: default
+          labels:
+            name: gcp-vm-disk-loss-by-label-sa
+            app.kubernetes.io/part-of: litmus
+        ---
+        apiVersion: rbac.authorization.k8s.io/v1
+        kind: ClusterRole
```

Comment:
Do we need ClusterRole? I am assuming since this is a non-K8s experiment, ClusterRole might not be required in this case.

Reply:
You're right but the charts are yet to be updated hence the ClusterRole.